src/index.html

```diff
@@ -2,37 +2,58 @@
 layout: pageNoEdit
 permalink: /
 ---
+<div class="px-4 py-5 my-5 text-center" style="padding-bottom: 0 !important;">
```

Comment:
Best practice for styles should be in css/less file and applied by class rather than in style tag in HTML.